docs/src/pages/[platform]/components/ComponentsGrid.tsx

```diff
@@ -8,21 +9,22 @@ import {
   navigationComponents,
   utilityComponents,
 } from '@/data/links';
-import { Card, Flex, Grid, Heading } from '@aws-amplify/ui-react';
+
 import Link from 'next/link';
-import { useRouter } from 'next/router';
+import { useCustomRouter } from '../../../components/useCustomRouter';
```

Comment:
Non-blocking nit: this import could be simplified to `'@/components/useCustomRouter';`

Reply:
done!